SCClassLibrary/Common/Control/Server.sc

```diff
@@ -375,18 +377,27 @@ Server {
 	}
 
 	initTree {
-		this.newNodeAllocators;
 		this.sendDefaultGroups;
 		tree.value(this);
 		ServerTree.run(this);
 	}
 
 	/* id allocators */
 
-	// private, called from server notify response with next free clientID
+	clientIDLocked { ^this.serverRunning and: this.notified }
```

Comment:
IIRC, when you want to inline the test, you need to write `this.serverRunning and: { this.notified }`

Reply:
ok, done.